src/coins.ts

```diff
@@ -1274,6 +1274,17 @@ export const wrappedTokens: { [ChainId: string]: Token } = {
     name: 'WRAPPED ONE',
     logoURI: 'https://d1xrz6ki9z98vb.cloudfront.net/venomswap/tokens/WONE.png',
   },
+  [ChainId.ARBT]: {
+    // https://testnet.arbiscan.io/token/0x207ed1742cc0bebd03e50e855d3a14e41f93a461
+    address: '0x207ed1742cc0bebd03e50e855d3a14e41f93a461',
```

Comment:
I wondered why there only very few wallets holding the WETH token you linked.

When using https://bridge.arbitrum.io/ on testnet (rinkeby) an other WETH token is used, this one looks better:
https://testnet.arbiscan.io/token/0xb47e6a5f8b33b3f17603c83a0535a9dcd7e32681